warehouse/accounts/models.py

```diff
@@ -475,3 +480,42 @@ class ProhibitedUserName(db.Model):
     )
     prohibited_by: Mapped[User] = orm.relationship(User)
     comment: Mapped[str] = mapped_column(server_default="")
+
+
+class UniqueLoginStatus(str, enum.Enum):
+    PENDING = "pending"
+    CONFIRMED = "confirmed"
+
+
+class UserUniqueLogin(db.Model):
+    __tablename__ = "user_unique_logins"
+    __table_args__ = (
+        UniqueConstraint(
+            "user_id", "ip_address", name="_user_unique_logins_user_id_ip_address_uc"
+        ),
+    )
+
+    user_id: Mapped[UUID] = mapped_column(
+        PG_UUID(as_uuid=True),
+        ForeignKey("users.id", onupdate="CASCADE", ondelete="CASCADE"),
+        nullable=False,
+        index=True,
+    )
+    user: Mapped[User] = orm.relationship(back_populates="unique_logins")
+
+    ip_address: Mapped[str] = mapped_column(String, nullable=False)
```

Comment:
question: Should this be its own string, or a relationship to `warehouse.events.models.IpAddress`?